onboarding/source/controller.c

```diff
@@ -99,6 +117,8 @@ static void controllerTask(void * pvParameters) {
         /* USER CODE BEGIN */
         // Deal with error when initializing light service task and/or queue
 
+        sciPrintf("ERROR: Failed to initialize light service task and/or queue");
```

Comment:
For future reference, you should print a `\n` after these error messages so they're easier to view. Not a big deal tho, so dw about it for this challenge